src/Microsoft.Fx.Portability.MetadataReader/DependencyFinderEngineHelper.cs

```diff
@@ -198,5 +189,38 @@ private string GetPrefix(MemberReference memberReference)
                     return memberReference.GetKind().ToString();
             }
         }
+
+        /// <summary>
+        /// Tries to locate the assembly containing <see cref="System.Object"/>.
+        /// </summary>
+        private AssemblyReferenceInformation GetSystemRuntimeAssemblyInformation()
+        {
+            const string mscorlib = nameof(mscorlib);
+            const string SystemRuntime = "System.Runtime";
+
+            var microsoftAssemblies = _reader.AssemblyReferences
+                .Select(handle =>
+                {
+                    var assembly = _reader.GetAssemblyReference(handle);
+                    return _reader.FormatAssemblyInfo(assembly);
+                })
+                .Where(assembly => _assemblyFilter.IsFrameworkAssembly(assembly));
```

Comment:
you could just do `.Where(_assemblyFIlter.IsFrameworkAssembly)`

Reply:
Thanks! I always forget you can do that.